src/dwarf2pdb.cpp

```diff
@@ -1476,10 +1480,54 @@ bool CV2PDB::createTypes()
 				break;
 
 			case DW_TAG_subprogram:
-				if (id.name && id.pclo && id.pchi)
+				if (id.name)
 				{
-					addDWARFProc(id, cu, cursor.getSubtreeCursor());
-					int rc = mod->AddPublic2(id.name, img.codeSegment + 1, id.pclo - codeSegOff, 0);
+					if (!id.is_artificial)
+					{
+						unsigned long entry_point = 0;
+						if (id.pcentry)
+						{
+							entry_point = id.pcentry;
+						}
+						else if (id.pclo)
+						{
+							entry_point = id.pclo;
+						}
+						else if (id.ranges != ~0)
+						{
+							entry_point = ~0;

```

Comment:
Can't you use `0` here instead of `~0` here?

Reply:
`~0` is used here because we call `min` on `entry_point` later.  If we left it at `0` we'd have to check for `0` before calling `min`, so this seemed more convenient.

Reply:
Ok. I slightly lean towards the alternative (no intermediate modification of the variable), but it can be changed later if deemd necessary.